lib/livebook/boot.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Livebook.Boot do
```

Comment:
I think this is not used?

Reply:
Whoops, good catch, I created a more generic one!